assets/js/googlesitekit/data/create-settings-store.js

```diff
@@ -80,6 +82,7 @@ export const createSettingsStore = (
 		storeName = undefined,
 		settingSlugs = [],
 		initialSettings = undefined,
+		haveSettingsChanged = null,
```

Comment:
Let's call it `validateHaveSettingsChanged` to be consistent with other overrides.